go/vt/vtadmin/cluster/resolver/resolver.go

```diff
@@ -391,6 +391,7 @@ func (r *resolver) resolve() (*grpcresolver.State, error) {
 	defer cancel()
 
 	addrs, err := r.discoverAddrs(ctx, r.opts.DiscoveryTags)
+	log.Infof("Fetched discovery adresses for cluster %s and component %s: %+v", r.cluster, r.component, addrs)
```

Comment:
this is going to be pretty noisy, is there a debug level?

Reply:
Removed